### DISCUSSION TITLE
Effect Tweening

### DISCUSSION THREAD

**Discussion by Glebka73a1** [ntsc-rs/ntsc-rs#210] [Ideas]
Sometimes in order to create a realistic VHS tracking error video, constant effects values isn't enough. I think tweening for effects would be very important to create those, also would be nice if you were able to modify the tween style, like linear, exponential, sine, or create your own tween style preset. I hope this gets added in future updates.

**Comment by valadaptive** [comment]
Keyframing/tweening has been suggested before (see https://github.com/valadaptive/ntsc-rs/issues/67 and https://github.com/valadaptive/ntsc-rs/issues/134) and is too complex to implement in the standalone application. If you want to keyframe effect values, you can use the After Effects/Premiere or OpenFX plugins with video editing software. DaVinci Resolve is free and supports OpenFX plugins, and Vegas does too (although it is paid).
